src/tbp/monty/frameworks/config_utils/config_args.py

```diff
@@ -1153,6 +1154,189 @@ class FiveLMMontySOTAConfig(FiveLMMontyConfig):
+                    }
+                },
+                feature_weights={
+                    "patch": {
+                        "hsv": np.array([1, 0.5, 0.5]),
+                    }
+                }
+            }
+            ```
+          When we are constructing the config for learning module **i**, that entry
+          for `"patch"` will be replaced with `"patch_{i}"`. This is currently
+          done for three possible items in `learning_module_args`:
+          `"graph_delta_thresholds"`, `"tolerances"`, and `"feature_weights"`.
+
+
+    Args:
+        n_lms (int): Number of learning modules.
+        monty_class (type): Monty class.
+        learning_module_class (type): Learning module class.
+        learning_module_args (dict, optional): Arguments for learning modules.
+        sensor_module_class (type): Sensor module class.
+        sensor_module_args (dict, optional): Arguments for sensor modules.
+        motor_system_class (type): Motor system class.
+        motor_system_args (Mapping, optional): Arguments for motor system.
+        monty_args (Mapping, MontyArgs, optional): Arguments for monty.
+
+    Returns:
+        `MontyConfig`: complete monty config for multi-LM experiment.
+    """
+    # Make learning module configs.
+    if learning_module_args is None:
+        learning_module_args = {}
+    learning_module_configs = {}
+    for i in range(n_lms):
+        lm_args_i = copy.deepcopy(learning_module_args)
+        # Rename specs keyed with "patch" to "patch_{i}".
+        for name in ["graph_delta_thresholds", "tolerances", "feature_weights"]:
+            if name in lm_args_i:
+                spec = lm_args_i[name]
+                if "patch" in spec:
+                    spec[f"patch_{i}"] = spec.pop("patch")
+        learning_module_configs[f"learning_module_{i}"] = {
+            "learning_module_class": learning_module_class,
+            "learning_module_args": lm_args_i,
+        }
+
+    # Make sensor module configs.
+    if sensor_module_args is None:
+        sensor_module_args = {}
+    sensor_module_configs = {}
+    for i in range(n_lms):
+        sm_args_i = copy.deepcopy(sensor_module_args)
+        sm_args_i["sensor_module_id"] = f"patch_{i}"
+        sensor_module_configs[f"sensor_module_{i}"] = {
+            "sensor_module_class": sensor_module_class,
+            "sensor_module_args": sm_args_i,
+        }
+    sensor_module_configs["view_finder"] = {
+        "sensor_module_class": DetailedLoggingSM,
+        "sensor_module_args": {
+            "sensor_module_id": "view_finder",
+            "save_raw_obs": True,
+        },
+    }
+
+    # Make motor system config.
+    if motor_system_args is None:
+        motor_system_args = {}
+    else:
+        motor_system_args = copy.deepcopy(motor_system_args)
+    motor_system_config = {
+        "motor_system_class": motor_system_class,
+        "motor_system_args": motor_system_args,
+    }
+
+    conn = make_multi_lm_connectivity(n_lms)
```

Comment:
I would just call this variable `connectivity` or even `connectivity_matrices`. There is enough space to spell variables out for clarity

Reply:
Done.